packages/eslint-plugin/src/rules/no-input-prefix.ts

```diff
@@ -0,0 +1,85 @@
+import { TSESTree } from '@typescript-eslint/experimental-utils';
+import { createESLintRule } from '../utils/create-eslint-rule';
+import { getClassPropertyName, isImportedFrom } from '../utils/utils';
+
+type Options = [
+  {
+    prefixes: string[];
+  },
+];
+export type MessageIds = 'noInputPrefix';
+export const RULE_NAME = 'no-input-prefix';
+
+export default createESLintRule<Options, MessageIds>({
+  name: RULE_NAME,
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description:
+        'Input names should not be prefixed by the configured disallowed prefixes.',
+      category: 'Best Practices',
+      recommended: false,
+    },
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          prefixes: {
+            type: 'array',
+            items: {
+              type: 'string',
+            },
+          },
+        },
+        additionalProperties: false,
+      },
+    ],
+    messages: {
+      noInputPrefix: `@Inputs should not be prefixed by {{blacklistedPrefixes}}`,
```

Comment:
Please could we change this language to match what you say in the feedback - i.e. use `disallowed` everywhere, instead of `blacklist`. I know some people feel strongly about this terminology and it's very easy to change to match with the feedback language so we may as well

Reply:
Sounds good, thanks!